cristin-commons/src/main/java/no/unit/nva/model/Organization.java

```diff
@@ -13,17 +12,21 @@
 import java.util.Set;
 
 import static no.unit.nva.cristin.model.JsonPropertyNames.ACRONYM;
+import static no.unit.nva.cristin.model.JsonPropertyNames.CONTEXT;
+import static no.unit.nva.cristin.model.JsonPropertyNames.HAS_PART;
 import static no.unit.nva.cristin.model.JsonPropertyNames.ID;
 import static no.unit.nva.cristin.model.JsonPropertyNames.NAME;
 import static no.unit.nva.cristin.model.JsonPropertyNames.PART_OF;
 import static no.unit.nva.cristin.model.JsonPropertyNames.TYPE;
 
 
-@SuppressWarnings("PMD.BeanMembersShouldSerialize")
+@SuppressWarnings({"PMD.BeanMembersShouldSerialize", "PMD.LinguisticNaming"})
+@JsonPropertyOrder({CONTEXT, ID, TYPE, NAME, ACRONYM, PART_OF, HAS_PART})
 @JsonTypeInfo(use = JsonTypeInfo.Id.NAME, property = "type")
-@JsonPropertyOrder({ID, TYPE, NAME, ACRONYM, PART_OF})
 public class Organization implements JsonSerializable {
 
+    public static final String ORGANIZATION_IDENTIFIER_PATTERN = "^(?:[0-9]+\\.){3}[0-9]{1,3}$";
+    public static final String ORGANIZATION_CONTEXT = "https://api.dev.nva.aws.unit.no/cristin/organization";
```

Comment:
Is this a context uri or a regular uri?

Reply:
It's a contexturi waiting to be determined... Just a placeholder for now